gulpfile.js

```diff
@@ -440,6 +440,7 @@ function buildExtensionJs(path, name, version, options) {
     // See https://github.com/ampproject/amphtml/issues/3977
     wrapper: options.noWrapper ? '' : ('(self.AMP=self.AMP||[])' +
         '.push({n:"' + name + '",' + priority +
+        'v:"' + internalRuntimeVersion + '",' +
```

Comment:
Does this include RTV prefix? I always get confused where it is added and where it's not. Does it matter?